packages/react/runtime/src/opcodes.ts

```diff
@@ -75,10 +75,25 @@ export function ssrHydrateByOpcodes(
         break;
       }
       case Opcode.Attr: {
+        // On web-platform, we need to set the `__elements` and `__element_root` properties
+        // before `setAttribute`, because web-platform requires some PAPI to be called again like `__AddEvent`.
+        // TODO(hzy): Check if it is `__AddEvent`
+        // @ts-expect-error fixme: https://github.com/lynx-family/lynx/pull/1429
+        const isWeb = SystemInfo.platform === 'web';
```

Comment:
Consider moving this check to the module scope to prevent repeated evaluations for every `Opcode.Attr`.